### ISSUE TITLE
[Bug]: Fps Drops + Crashes

### ISSUE THREAD

**Issue by lewistrm** [TreyRuffy/BetterF3#89]
### Describe the Bug

when starting up the game fps already drops massively. i can work with that since i have a really good pc.

but as soon as you open f3 will all modules disabled on the right side, the game comes back with a "Error: java.lang.IllegalStateException: BufferBuilder was empty" crash

-edited

### Steps to Reproduce

disable all text on the right side of the screen 

the video below will show me get rid of all text on the right side by not targetting a block to make it clear that the crash occurs as soon as there is no text on the right side

-edited

### Expected Behavior

it will crash with Error: java.lang.IllegalStateException: BufferBuilder was empty, normally it shouldnt crash

### Screenshots

https://github.com/TreyRuffy/BetterF3/assets/42900973/36c77382-2033-4fa4-942b-519c2ca09901

### Modloader Type

Minecraft Forge

### Modloader Version

fabric 1.21

### Mod Version

BetterF3 Fabric v11.0.0 (1.21)

### Log Output

```shell
Error: java.lang.IllegalStateException: BufferBuilder was empty
```

**Comment by lewistrm** [comment]
A little extra info: i took out all my other mods out of the mods folder and left in just betterf3 and cloth config, still the same issue. so its not mods colliding with each other

**Comment by lewistrm** [comment]
even more info that ive gathered. it crashed when you arent aiming at a block within the f3 menu

**Comment by lewistrm** [comment]
even more info that ive gathered: the crash is caused by having 0 text showing on the right side of the screen in any form

tested this further, this is definitely causing the crash. in the video i sent above it crashed because i stopped aiming at a block which removed the text from the module off of the right side which crashes the game due to no text on that side. i have temporarily fixed the issue by adding 1 line of text through a different module and it doesnt crash now. but i urge you to fix this, this only applies to the 1.21 version

**Comment by DatapackMaster** [comment]
I experience the same issue

**Comment by Fabsian10000** [comment]
same thing here

**Comment by TreyRuffy** [comment]
Fixed in 11.0.1
